arduino/WandCore/WandCore.ino

```diff
@@ -99,19 +100,19 @@ RemoteDebug Debug;
 ////////////////////////////////////////////////////////////////////////////////////
 // Set up state
 ////////////////////////////////////////////////////////////////////////////////////
-int wandMode = 1; // 1 = power/brightness, 2 = color
-int rainbowSection = 1; // this holds the state of where in the 6 rainbow sections the colorloop is
-int rainbowVar = 0; // this holds the state of the rgb colorloop variable
+// wand mode definitions
+#define POWER 0
+#define COLOR 1

```

Comment:
Usually definitions are put at the top of a file, or in a header file, but I don't think there's necessarily anything wrong with doing it here. Just noting.

Reply:
ok - i'm going to keep it here because I like the logical grouping.